package.json

```diff
@@ -28,14 +37,16 @@
     "url": "git+https://github.com/google/postcss-rename.git"
   },
   "dependencies": {
-    "postcss-selector-parser": "^7.1.0"
+    "postcss-selector-parser": "^7.1.0",
+    "postcss-value-parser": "^4.2.0"
   },
   "devDependencies": {
     "@types/jest": "^26.0.24",
     "@types/node": "^12.20.55",
     "gts": "^6.0.2",
     "jest": "^29.7.0",
     "jest-cli": "^29.7.0",
+    "npm-dts": "^1.3.13",
```

Comment:
Per offline conversation, we shouldn't use `npm-dts`.

Instead, users of this library must use `module: node16` and `moduleResolution: node16` in their `tsconfig` file.

```suggestion
```